subsys/bluetooth/host/keys.h

```diff
@@ -7,15 +7,15 @@
  */
 
 enum {
-	BT_KEYS_SLAVE_LTK      = BIT(0),
+	BT_KEYS_PERIPH_LTK     = BIT(0),
```

Comment:
Didn't we agree not to shorten "peripheral" in the other PR?

Reply:
But those are internal macros that already shorten much more than the public ones. If you prefer I can spell it out, but this is a different case IMHO.

Reply:
I do prefer the full spelling, yes, but it is not something I'll block this PR with. Since it has already been approved by several others, I suppose they are also fine with this naming.